src/viam/ur/module/ur_arm_state_disconnected.cpp

```diff
@@ -12,23 +12,53 @@ std::string URArm::state_::state_disconnected_::describe() const {
 }
 
 std::chrono::milliseconds URArm::state_::state_disconnected_::get_timeout() const {
-    return std::chrono::seconds(1);
+    // If we have a pending connection, we are polling for completion,
+    // so increase the sampling rate. Otherwise we want to wait a more
+    // reasonable length of time before attempting to connect again.
+    return pending_connection ? std::chrono::milliseconds(5) : std::chrono::seconds(1);
```

Comment:
while the arm is turned off, do we have an idea of how long we are in the pending_connection state? 

this makes sense because we only try connecting to the arm at most once per second, but I don't remember how long connection attempts would take(seemingly long because we are doing this ticket)

Reply:
It isn't so much that they are long, exactly. More that we were holding the state machine lock while we were doing it, which made it impossible to answer queries about our state (which, naturally, must also take the lock so they can observe stable values as updated by the running state machine).